scripts/reindex.py

```diff
@@ -70,6 +71,10 @@ def __init__(self, prog) -> None:
                     default=False,
                     action='store_true',
                     help='Purge the queues before taking any action on the index.')
+parser.add_argument('--nowait', '--no-wait',
```

Comment:
Use `default=True, dest='wait', action='store_false'` to prevent the negation from leaking.